src/routes/Team/Team.tsx

```diff
@@ -8,25 +8,23 @@ import TeamMember from '@/types/TeamMember'
 import TeamMemberCard from '@/components/TeamMemberCard/TeamMemberCard'
 import team from './team.json'
 
-
 const Team: FC = () => {
     const { t } = useTranslation()
 
-    const teamGrid: ReactNode[] = []
-    team.forEach((member: TeamMember) => {
-        teamGrid.push(<Grid key={member.nameEN}>
-            <TeamMemberCard member={member} />
-        </Grid>)
-    })
-
-    return <Container style={{ padding: 32 }} aria-label="team-container">
-        <Stack spacing={2}>
-            <Typography variant="h1">{t('team.title')}</Typography>
-            <Grid container spacing={2}>
-                {teamGrid}
-            </Grid>
-        </Stack>
-    </Container>
+    return (
+        <Container className="section-padding" aria-label="team-container">
+            <Stack spacing={2}>
+                <Typography variant="h1">{t('team.title')}</Typography>
```

Comment:
As a rule, we like to keep one change per PR in this codebase for simplicity. So if you'd like to do other refactors, we'd welcome a separate PR, but combining a bunch of changes at once makes it hard to tell what is happening.